src/main/java/com/aws/greengrass/certificatemanager/CertificateManager.java

```diff
@@ -106,7 +109,8 @@ public void stopMonitors() {
      * @throws CertificateEncodingException if unable to get certificate encoding
      */
     public List<String> getCACertificates() throws KeyStoreException, IOException, CertificateEncodingException {
-        return Collections.singletonList(CertificateHelper.toPem(certificateStore.getCACertificate()));
+        return Collections.singletonList(EncryptionUtils.encodeToPem(PEM_BOUNDARY_CERTIFICATE,
```

Comment:
This outputs precisely the same thing? Even if it does, I wouldn't want you to change this unless there's a reason.

Reply:
I'll keep `CertificateHelper.toPem` and revert then.